seqr/views/apis/family_api_tests.py

```diff
@@ -20,9 +23,83 @@
 FAMILY_ID_FIELD = 'familyId'
 PREVIOUS_FAMILY_ID_FIELD = 'previousFamilyId'
 
-
 class FamilyAPITest(AuthenticationTestCase):
-    fixtures = ['users', '1kg_project']
+    fixtures = ['users', '1kg_project', 'reference_data']
+
+    @mock.patch('seqr.views.utils.permissions_utils.ANALYST_PROJECT_CATEGORY', 'analyst-projects')
+    @mock.patch('seqr.views.utils.orm_to_json_utils.ANALYST_USER_GROUP', 'analysts')
+    @mock.patch('seqr.views.utils.permissions_utils.ANALYST_USER_GROUP')
+    def test_family_page_data(self, mock_analyst_group):
+        url = reverse(family_page_data, args=[FAMILY_GUID])
+        self.check_collaborator_login(url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+        response_json = response.json()
+        response_keys = {
+            'familiesByGuid', 'individualsByGuid', 'familyNotesByGuid', 'samplesByGuid',  'igvSamplesByGuid',
+            'mmeSubmissionsByGuid',
+        }
+        self.assertSetEqual(set(response_json.keys()), response_keys)
+
+        family = response_json['familiesByGuid'][FAMILY_GUID]
+        family_fields = {'individualGuids', 'detailsLoaded'}
+        family_fields.update(FAMILY_FIELDS)
+        self.assertSetEqual(set(family.keys()), family_fields)
+
+        individual_fields = {'sampleGuids', 'igvSampleGuids', 'mmeSubmissionGuid'}
+        individual_fields.update(INDIVIDUAL_FIELDS)
+        self.assertSetEqual(set(next(iter(response_json['individualsByGuid'].values())).keys()), individual_fields)
+        self.assertSetEqual(set(next(iter(response_json['samplesByGuid'].values())).keys()), SAMPLE_FIELDS)
+        self.assertSetEqual(set(next(iter(response_json['familyNotesByGuid'].values())).keys()), FAMILY_NOTE_FIELDS)
+        self.assertSetEqual(set(next(iter(response_json['igvSamplesByGuid'].values())).keys()), IGV_SAMPLE_FIELDS)
+        self.assertSetEqual(
+            set(next(iter(response_json['mmeSubmissionsByGuid'].values())).keys()), MATCHMAKER_SUBMISSION_FIELDS
+        )
```

Comment:
yeah, this was a sort of copy paste from the project details test since that was the endpoint that was refactored, and those tests didn't test the content. But you are right that it would be better to